README.md

```diff
@@ -1,45 +1,76 @@
 # Sentry Release GitHub Action
-Automatically create a Sentry Release in a workflow.
-
-## Why Releases?
-A release is a version of your code that is deployed to an environment. 
-When you give Sentry information about your releases, you unlock a number of new features:
- - Determine the issues and regressions introduced in a new release
- - Predict which commit caused an issue and who is likely responsible
- - Resolve issues by including the issue number in your commit message
- - Receive email notifications when your code gets deployed
-
-Additionally, releases are used for applying source maps to minified JavaScript to view original, untransformed source code.
+Automatically create a Sentry release in a workflow. 
```

Comment:
Think we should explain why creating releases, deploys, sending commit metadata to Sentry is valuable